runtime/near-vm-runner/Cargo.toml

```diff
@@ -34,6 +34,7 @@ default = [ "wasmer_default"]
 wasmtime_vm = [ "wasmtime", "anyhow"]
 wasmer_default = []
 wasmtime_default = ["wasmtime_vm"]
+lightbeam = ["wasmtime_default"]
```

Comment:
Hmm, not sure what is intended behavior here. wasmtime_default is feature enabling wasmtime by default, lightbeam shall depend on `wasmtime_vm`.

Reply:
Yeah actually the only place where it changes the default is in vm-logic.  So this depends on the `wasmtime_default` above which in turn depends on `wasmtime_vm` and so on.  Should I change it to `wasmtime_vm` so that this is not an issue?